tox.cover.sh

```diff
@@ -37,7 +42,7 @@ cover () {
   # specific package directory; --cover-tests makes sure every tests
   # is run (c.f. #403)
   nosetests -c /dev/null --with-cover --cover-tests --cover-package  \
-            "$1" --cover-min-percentage="$min" "$1"
+            "$1" --cover-min-percentage="$min" "$1" $proc
```

Comment:
nit: Since all packages use the same number of processes, why not just put that down here instead of setting it to the same thing in each if block? In the if block, you can just set a `timeout` value, which defaults to 10 for `nosetests`.